test/templates/pagination/component.njk

```diff
@@ -0,0 +1,35 @@
+{#
+    see ./data.json for object structure
+#}
+
+{% macro pagination(properties={}, modifier={}, state={}, aria={}) -%}
+<ul class="tn-pagination">
+    <li class="tn-pagination--link-wrapper">
+      <a href="#" class="tn-pagination--link" aria-label="Previous">
+        <span aria-hidden="true"><<</span>
+      </a>
+    </li>
+    <li class="tn-pagination--link-wrapper">
+        <a href="#" class="tn-pagination--link">1</a>
+    </li>
+    <li class="tn-pagination--link-wrapper">
+        <a href="#" aria-selected="true" class="tn-pagination--link">2</a>
+    </li>
+    <li class="tn-pagination--link-wrapper">
+        <a href="#" class="tn-pagination--link">3</a>
+    </li>
+    <li class="tn-pagination--link-wrapper">
+        <a href="#" class="tn-pagination--link tn-pagination--more" aria-label="Next">
```

Comment:
You don't need a link here at all. This should be plain text inside the `__item`. The label is also incorrect.